reference-implementation/lib/readable-stream.js

```diff
@@ -859,7 +859,10 @@ class ReadableStreamDefaultController {
 
     this._underlyingSource = underlyingSource;
 
-    this._queue = [];
+    // Need to set the slots so that the assert doesn't fire. In the spec the slots already exist implicitly.
+    this._queue = this._queueTotalSize = undefined;
```

Comment:
Optional: I'd prefer not to use chained assignment. As far as I can tell the rest of the reference implementation sticks to one assignment per statement.